functionary/ui/templates/admin/constance/includes/results_list.html

```diff
@@ -0,0 +1,63 @@
+{% load admin_list static i18n %}
+<div id="results">
+    <table>
+        <thead>
+            <tr>
+                <th>
+                    <div class="text">{% trans "Name" %}</div>
+                </th>
+                <th>
+                    <div class="text">{% trans "Default" %}</div>
+                </th>
+                <th>
+                    <div class="text">{% trans "Value" %}</div>
+                </th>
+                <th>
+                    <div class="text">{% trans "Is modified" %}</div>
+                </th>
+            </tr>
+        </thead>
+        {% for item in config_values %}
+            <tr class="{% cycle 'row1' 'row2' %}">
+                <th>
+                    <span class="item-name" id="{{ item.name|slugify }}">
+                        {% if item.help_text %}
+                            {{ item.help_text }}
+                        {% else %}
+                            {{ item.name }}
+                        {% endif %}
+                        <a class="item-anchor"
+                           href="#{{ item.name|slugify }}"
+                           title="Link to this setting">¶</a>
+                    </span>
+                </th>
+                <td>{{ item.default|linebreaks }}</td>
+                <td>
+                    {{ item.form_field.errors }}
+                    {% if item.is_file %}
+                        {% trans "Current file" %}: <a href="{% get_media_prefix as MEDIA_URL %}{{ MEDIA_URL }}{{ item.value }}"
+    target="_blank">{{ item.value }}</a>
+                    {% endif %}
+                    {{ item.form_field }}
+                    {% if not item.is_file %}
+                        <br/>
```

Comment:
Just a note of some css fix to come back to: The padding on the <p> elements in this particular table cell causes an out-of-place indentation. IMO it would look better if there was no indentation, so that it aligns with the left-alignment like the other cells